web/src/stream_settings_ui.js

```diff
@@ -443,8 +445,25 @@ export function render_left_panel_superset() {
     ui.get_content_element($("#manage_streams_container .streams-list")).html(html);
 }
 
+export function render_empty_left_panel_message() {
+    const is_empty = $("#manage_streams_container .stream-row:not(.notdisplayed)").length === 0;
+    if (!is_empty) {
+        $(".no-streams-to-show").hide();
+        return;
+    }
+    if (is_subscribed_stream_tab_active()) {
+        $(".all_streams_tab_empty_text").hide();
+        $(".subscribed_streams_tab_empty_text").show();
+    } else {
+        $(".subscribed_streams_tab_empty_text").hide();
+        $(".all_streams_tab_empty_text").show();
+    }
+    $(".no-streams-to-show").show();
+}
+
 // LeftPanelParams { input: String, subscribed_only: Boolean, sort_order: String }
 export function redraw_left_panel(left_panel_params = get_left_panel_params()) {
+    $(".no-streams-to-show").hide();
```

Comment:
Same question as asked before. We would be hiding the element if required below in `render_empty_left_panel_message`. Is this added for not showing the message while streams are loading or something like that.

Reply:
This is strange, as I have done this change and pushed the code but it is same here, also in my local clone there is this change exists and git is showing this message - `Your branch and 'origin/issue-21870' have diverged,
and have 1 and 1 different commits each, respectively.` . I remember I was getting some error without any details while pushing the code than I ran `git gc` command as suggested on an stackoverflow solution and than pushed the code successfully, could it be a reason for this buggy experience ?

Reply:
I am not sure why this happened. Cannot tell something without knowing the error.